packages/features/embed/Embed.tsx

```diff
@@ -418,7 +418,13 @@ const EmailEmbedPreview = ({
                                   <tr style={{ height: "25px" }}>
                                     {selectedDateAndTime[key]?.length > 0 &&
                                       selectedDateAndTime[key].map((time) => {
-                                        const bookingURL = `${WEBSITE_URL}/${username}/${eventType.slug}?duration=${eventType.length}&date=${key}&month=${month}&slot=${time}`;
+                                        // If teamId is present on the eventType object, it means that this is a team event
+                                        // And hence we add 'team/' to the bookingUrl.
+                                        const bookingURL = `${WEBSITE_URL}/${
```

Comment:
Nice find, can you create a new PR with only fixing this? 

We just approved another PR for the webhook fix: https://github.com/calcom/cal.com/pull/14922

Reply:
Yes sure. I have already created a PR addressing this issue.
**PR: #14889**
Please have a look at it. Also I'm sorry, I've committed yarn.lock there also. I'm new to opensource.